modules/logging-loki-cli-install.adoc

```diff
@@ -0,0 +1,188 @@
+
+. Install the `Red Hat OpenShift Logging` Operator by creating the following objects:
+
+.. Create an Operator Group object YAML file (for example, `olo-og.yaml`) for the Red Hat OpenShift Logging Operator using the template below:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging <1>
+spec:
+  targetNamespaces:
+  - openshift-logging <1>
+----
+<1> You must specify the `openshift-logging` namespace.
+
+
+.. Create a Subscription object YAML file (for example, `olo-sub.yaml`) to
+subscribe a namespace to the Red Hat OpenShift Logging Operator using the template below:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging <1>
+spec:
+  channel: "stable" <2>
+  name: cluster-logging
+  source: redhat-operators <3>
+  sourceNamespace: openshift-marketplace
+----
+<1> You must specify the `openshift-logging` namespace.
+<2> Specify `stable`, or `stable-5.<y>` as the channel.
+<3> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the CatalogSource object you created when you configured the Operator Lifecycle Manager (OLM).
+
++
+--
+include::snippets/logging-stable-updates-snip.adoc[]
+--
+
+.. Verify the Operator installation.
++
+There should be a Red Hat OpenShift Logging Operator in the `openshift-logging` namespace. The Version number might be different than what is shown.
++
+[source,terminal]
+----
+$ oc get csv -n openshift-logging
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE                                               NAME                                         DISPLAY                  VERSION               REPLACES   PHASE
+...
+openshift-logging                                       clusterlogging.5.5.0-202007012112.p0         OpenShift Logging          5.1.0-202007012112.p0              Succeeded
+...
+----
+
+. Create an OpenShift Logging instance:
+
+.. Create an instance object YAML file (for example, `olo-instance.yaml`) for the Red Hat OpenShift Logging Operator using the template below:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+--
+include::snippets/logging-clusterlogging-cr-snip.adoc[]
+--
+
+. Verify the installation by listing the pods in the *openshift-logging* project.
++
+You should see several pods for components of the Logging subsystem, similar to the following list:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-logging
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                            READY   STATUS    RESTARTS   AGE
+cluster-logging-operator-66f77ffccb-ppzbg       1/1     Running   0          7m
```

Comment:
I can't access the CL link on here but it should have default logstore as Loki.
Also plugin-pod needs to be here as well. 
`logging-view-plugin-547986d85c-hxff2`